src/Ocelot/Configuration/Repository/FileConfigurationRepository.cs

```diff
@@ -18,7 +18,7 @@ public FileConfigurationRepository(IHostingEnvironment hostingEnvironment)
             _configFilePath = $"{AppContext.BaseDirectory}/configuration{(string.IsNullOrEmpty(hostingEnvironment.EnvironmentName) ? string.Empty : ".")}{hostingEnvironment.EnvironmentName}.json";
         }
 
-        public async Task<Response<FileConfiguration>> Get()
+        public Task<Response<FileConfiguration>> Get()

```

Comment:
This method we should probably make 

```csharp
jsonConfiguration = System.IO.File.ReadAllText(_configFilePath);
```

into
```csharp
jsonConfiguration = System.IO.File.ReadAllTextAsync(_configFilePath);
```

Then we dont need the Task.FromResult